nautobot/dcim/forms.py

```diff
@@ -5298,6 +5298,11 @@ class VirtualDeviceContextForm(NautobotModelForm):
         required=True,
         query_params={"content_types": VirtualDeviceContext._meta.label_lower},
     )
+    vrfs = DynamicModelMultipleChoiceField(
```

Comment:
Do we want an add/remove vrfs option on the corresponding bulk-edit form?

Reply:
Device/VM BulkEditForm are missing an add/remove vrfs option as well. Do you want me to add those in this PR or follow up issue is fine?

Reply:
Let's get a followup issue for those just to keep the scope manageable here.

Reply:
https://github.com/nautobot/nautobot/issues/6938